source/docs/articles/users/access-management.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Access Management for Site Owners and Administrators
+description: Learn how to remove a team member or outside resource from a site.
+keywords: remove team member, delete team member, delete user
+---
+
+When a resource leaves the company or project, it is important to immediately remove them from the team  so that they no longer have access to make changes to your site. Pantheon makes it easy to manage your team, so you do not have to do any work within the OS or change users and passwords in the database.
```

Comment:
I'm not sure "you do not have to...change users and passwords in the database"  is accurate. Part of this process should be removing/blocking Drupal or WordPress user accounts. This could also be an opportunity to link out to "Secure Your Site with Two-Factor Authentication" https://pantheon.io/docs/guides/two-factor-authentication/